python/ray/data/tests/test_dataset.py

```diff
@@ -2964,83 +2968,6 @@ def range(n, parallelism=200):
     assert r1.take() == ds.take()
 
 
-@pytest.mark.parametrize("use_spread_resource_prefix", [False, True])
-def test_random_shuffle_spread(ray_start_cluster, use_spread_resource_prefix):
-    cluster = ray_start_cluster
-    cluster.add_node(
-        resources={"bar:1": 100},
-        num_cpus=10,
-        _system_config={"max_direct_call_object_size": 0},
-    )
-    cluster.add_node(resources={"bar:2": 100}, num_cpus=10)
-    cluster.add_node(resources={"bar:3": 100}, num_cpus=0)
-
-    ray.init(cluster.address)
-
-    @ray.remote
-    def get_node_id():
-        return ray.get_runtime_context().node_id.hex()
-
-    node1_id = ray.get(get_node_id.options(resources={"bar:1": 1}).remote())
-    node2_id = ray.get(get_node_id.options(resources={"bar:2": 1}).remote())
-
-    ds = ray.data.range(100, parallelism=2).random_shuffle(
-        _spread_resource_prefix=("bar:" if use_spread_resource_prefix else None)
-    )
-    blocks = ds.get_internal_block_refs()
-    ray.wait(blocks, num_returns=len(blocks), fetch_local=False)
-    location_data = ray.experimental.get_object_locations(blocks)
-    locations = []
-    for block in blocks:
-        locations.extend(location_data[block]["node_ids"])
-    assert set(locations) == {node1_id, node2_id}
-
-
-@pytest.mark.parametrize("use_spread_resource_prefix", [False, True])
-def test_parquet_read_spread(ray_start_cluster, tmp_path, use_spread_resource_prefix):
```

Comment:
We should still keep these two tests for "use_spread_resource_prefix" false case (i.e. using "SPREAD" scheduling strategy).